.env.example

```
@@ -2,4 +2,4 @@
 OCTAGON_API_KEY=your_octagon_api_key_here
 
 # Base URL for Octagon API
-OCTAGON_API_BASE_URL=https://api.octagonagents.com/v1 
\ No newline at end of file
+OCTAGON_API_BASE_URL=https://api.octagonai.co/v1 
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)